upp-pub-provisioner/ansible/userdata/persistent_instance_user_data.yaml

```diff
@@ -55,44 +55,46 @@ coreos:
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-mcpm-validation-url '$pam_mcpm_validation_url'                        >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-mimm-validation-url '$pam_mimm_validation_url'                        >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-ml-validation-url '$pam_mlm_validation_url'                           >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-maicm-validation-url '$pam_maicm_validation_url'                      >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-video-validation-url '$pam_video_validation_url'                      >/dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-wam-validation-url '$pam_wam_validation_url'                          >/dev/null 2>&1 || true;"
-          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-credential-validation-uuid '$pam_credential_validation_uuid'            >/dev/null 2>&1 || true;"
-          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/synthetic-article-uuid '$synthetic_article_uuid'                            >/dev/null 2>&1 || true;"
-          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/synthetic-article-payload '$synthetic_article_payload'                      >/dev/null 2>&1 || true;"
-          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/synthetic-list-uuid '$synthetic_list_uuid'                                  >/dev/null 2>&1 || true;"
-          ExecStart=/bin/sh -c "etcdctl mk /ft/_credentials/publish-read/read-credentials '$delivery_clusters_http_credentials'              >/dev/null 2>&1 || true;"
-          ExecStart=/bin/sh -c "etcdctl mk /ft/_credentials/publish-read/validator-credentials '$pam_validator_credentials'             >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/pam-credential-validation-uuid '$pam_credential_validation_uuid'          >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/synthetic-article-uuid '$synthetic_article_uuid'                          >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/synthetic-article-payload '$synthetic_article_payload'                    >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/monitoring/synthetic-list-uuid '$synthetic_list_uuid'                                >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/publish-carousel/bucket '$carousel_bucket'                                           >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/publish-carousel/enable '$carousel_enabled'                                          >/dev/null 2>&1 || true;"
```

Comment:
The above two lines are the only lines *added* here, the rest have just had their whitespace changed to make them lineup